python/containers/fetch-functions.sh

```diff
@@ -0,0 +1,19 @@
+export THIEF_KEYVAULT_NAME="thief-kv"
+
+echo "Getting token"
+while [ "$token" == "" ]; do 
+    response=$(curl --silent 'http://169.254.169.254/metadata/identity/oauth2/token?api-version=2018-02-01&resource=https%3A%2F%2Fvault.azure.net' -H Metadata:true)
```

Comment:
hard coded IP address?

Reply:
Yes.  169.254.169.254 is a well-known IP used to get identity data from Azure.  see [https://docs.microsoft.com/en-us/azure/virtual-machines/windows/instance-metadata-service](https://docs.microsoft.com/en-us/azure/virtual-machines/windows/instance-metadata-service) 

---
In reply to: [492338139](https://github.com/Azure/iot-sdk-longhaul/pull/9#discussion_r492338139) [](ancestors = 492338139)

Reply:
Worth a code comment, or better yet an aptly named variable then